raycicmd/wanda.go

```diff
@@ -61,9 +59,14 @@ func (s *wandaStep) buildkiteStep() map[string]any {
 	if s.dependsOn != nil {
 		bkStep["depends_on"] = s.dependsOn
 	}
-	if agent != "" {
-		bkStep["agents"] = newBkAgents(agent)
+
+	agentQueue := builderAgent(s.ciConfig, instanceType)
+	if agentQueue == skipQueue {
```

Comment:
where is this `skipQueue` defined?

Reply:
https://github.com/ray-project/rayci/blob/main/raycicmd/config.go#L12

it is already used for runners. this PR adds support for builders.